Add get ready post (#2205)
writethedocs/www: docs/conf/atlantic/2024/news/get-ready.rst

If you haven't purchased your ticket yet we invite you to `get your tickets soon <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.

For those of you who already purchased your tickets, you should have already received an email with our online conference platform (Venueless) registration link. We will also send a reminder on September 20th, in case you missed the first email, or registered later.
If you can not find this mail, please contact us on {{shortcode}}@writethedocs.org.

Welcome Wagon guide
-------------------

Each year we create a guide to help our attendees get the most out of the conference.
Head over to the website and check out the `Welcome Wagon guide <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/welcome-wagon/>`_ to help you get the most out of our event this year.

All our events and activities
-----------------------------

There are many events at the conference beyond the main track of speakers.
We hope that you'll get involved with some of our other events,
and get to know more folks that are hanging out at the conference!

> 

Reviewer: @mxsasha Does it make sense to have Writing Day here with a link to the event and a note that they are welcome to bring a project to announce day of?

Author: Yes, it belongs in there I think. Will add.

:template: {{year}}/generic.html

.. post:: Sept 16, 2024
   :tags: {{year}}, {{shortcode}}-{{year}}, tickets

Get Ready for the conference!
=============================

Hey everyone,

> We're just a few days from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.

Reviewer 2: ```suggestion
We're just a few days from the start of the conference, and we're getting excited to see you soon! We're looking forward to gathering for three days of content, collaboration, and conversation.
```

We have a few reminders before we welcome you all on Sunday:

* Ticket sales close this week, so grab yours now!

> * There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prep a bit for them.

Reviewer 2: ```suggestion
* There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prepare for them if you'd like to.
```
